tests/fixtures/elevation.yml

```diff
@@ -1,24 +1,24 @@
 http_interactions:
 - request:
     method: get
-    uri: http://api.geonames.org/srtm3?lats=32.909436%2C38.540315%2C37.050632%2C31.913204%2C32.648646%2C32.649972%2C47.42263%2C30.060134%2C37.04786%2C40.363404%2C47.538867%2C38.481486%2C37.050542%2C25.576582%2C31.334842%2C33.124711%2C33.068336%2C32.745623%2C41.695009%2C16.056195&lngs=-116.545853%2C-122.970018%2C-121.4302%2C-109.931659%2C-108.536463%2C-108.537201%2C-113.670813%2C-103.558096%2C-121.42573%2C-120.622698%2C-113.742705%2C-122.915802%2C-121.429824%2C-80.689018%2C-110.332383%2C-108.568249%2C-116.333314%2C-116.570241%2C-103.712241%2C-93.477916&username=<geonames_user>
+    uri: http://api.geonames.org/srtm3?lats=32.909436%2C38.540315%2C37.050632%2C31.913204%2C32.648646%2C32.649972%2C47.42263%2C30.060134%2C37.04786%2C40.363404%2C47.538867%2C38.481486%2C37.050542%2C25.576582%2C31.334842%2C33.124711%2C33.068336%2C32.745623%2C41.695009%2C16.056195&lngs=-116.545853%2C-122.970018%2C-121.4302%2C-109.931659%2C-108.536463%2C-108.537201%2C-113.670813%2C-103.558096%2C-121.42573%2C-120.622698%2C-113.742705%2C-122.915802%2C-121.429824%2C-80.689018%2C-110.332383%2C-108.568249%2C-116.333314%2C-116.570241%2C-103.712241%2C-93.477916&username=<gbif_user>
     body:
       encoding: ''
       string: ''
     headers:
       Accept-Encoding: gzip, deflate
       Accept: application/json, text/xml, application/xml, */*
-      User-Agent: r-curl/4.3 crul/1.0.0 rOpenSci(rgbif/3.4.0.95)
-      X-USER-AGENT: r-curl/4.3 crul/1.0.0 rOpenSci(rgbif/3.4.0.95)
+      User-Agent: r-curl/4.3.2 crul/1.2.0 rOpenSci(rgbif/3.6.1.99)
+      X-USER-AGENT: r-curl/4.3.2 crul/1.2.0 rOpenSci(rgbif/3.6.1.99)
```

Comment:
I think with an approach like https://medium.com/@george.shuklin/tips-and-tricks-on-http-s-session-recording-4194f99adbf#b4a0 you can exclude irrelevant headers that change all the time (User-Agent, Date, Server).

Reply:
Are you suggesting that the user-agent should not include version number or just not in the tests?